src/components/App/index.js

```diff
@@ -98,28 +77,64 @@ export const App = ({ options }) => {
       ? switchToHall()
       : switchToRules();
   }
-
+  
+  let nextQuestion = {}; 
+  const arrGameOverResuls = [];
+  let currentPlayerAnswer = "";
+  let quiz = {};
   //get data from API based on active game mode
   const apiDataFetcher = new ApiDataFetcher(options.swApiBaseUrl);
   playTheGame.addEventListener('click', () => play(config.selectedGame));
 
   async function play(gameMode) {
-    const quiz = new GameEngine(gameMode, apiDataFetcher);
+    quiz = new GameEngine(gameMode, apiDataFetcher);
 
     await quiz.fetchAllQuestionsForMode(gameMode);
 
     const gameView = new GameView();
-    const nextQuestion = quiz.generateNextQuestion();
+    nextQuestion = quiz.generateNextQuestion();
     gameView.displayQuestion(nextQuestion);
+ 
   }
+  
+
+  inGameMode.addEventListener('click', (event) => {
```

Comment:
Nie bardzo rozumiem, dlaczego ten handler jest podpięty akurat pod ten element. Zrobiłbym z tego funkcje i przekazał ją do komponentu game view, ale to możemy się zdzwonić i przegadać

Reply:
do przegadania na callu

Reply:
wrzuciłam funkcję handleAnswerSelected i przekazałam do gameView, walczę jeszcze z komponentem do ustawiania poziomu trudności